src/Sylius/Behat/Context/Api/Admin/ManagingCountriesContext.php

```diff
@@ -0,0 +1,379 @@
+    }
+
+    /**
+     * @When I enable it
+     */
+    public function iEnableIt(): void
+    {
+        $this->client->addRequestData('enabled', true);
+    }
+
+    /**
+     * @When I disable it
+     */
+    public function iDisableIt(): void
+    {
+        $this->client->addRequestData('enabled', false);
+    }
+
+    /**
+     * @When I save my changes
+     * @When I try to save changes
+     */
+    public function iSaveMyChanges(): void
+    {
+        $this->client->update();
+    }
+
+    /**
+     * @Then I should be notified that it has been successfully edited
+     */
+    public function iShouldBeNotifiedThatItHasBeenSuccessfullyEdited(): void
+    {
+        Assert::true(
+            $this->responseChecker->isUpdateSuccessful($this->client->getLastResponse()),
+            'Country could not be edited'
+        );
+    }
+
+    /**
+     * @Then /^(this country) should be ([^"]+)$/
+     */
+    public function thisCountryShouldBeDisabled(CountryInterface $country, string $enabled): void
+    {
+        Assert::true(
+            $this->responseChecker->hasValue(
+                $this->client->show($country->getCode()),
+                'enabled',
+                $enabled === 'enabled' ? true : false
+            ),
+            'Country is not disabled'
+        );
+    }
+
+    /**
+     * @Then the code field should be disabled
+     */
+    public function theCodeFieldShouldBeDisabled(): void
+    {
+        /** @var CountryInterface $country */
+        $country = $this->sharedStorage->get('country');
+
+        $countryUpdateSerialised = $this->serializer->serialize($country, 'json', ['groups' => 'country:update']);
+        Assert::keyNotExists(\json_decode($countryUpdateSerialised, true), 'code');
+    }
+
+    /**
+     * @When I name the province :provinceName
+     */
+    public function iNameTheProvince(string $provinceName): void
+    {
+        $this->client->addSubResourceData(
+            'provinces',
+            ['name' => $provinceName]
+        );
+    }
+
+    /**
+     * @When I specify the province code as :provinceCode
+     */
+    public function iSpecifyTheProvinceCodeAs(string $provinceCode): void
+    {
+        $this->client->addSubResourceData(
+            'provinces',
+            ['code' => $provinceCode]
+        );
+    }
+
+    /**
+     * @When I add the :provinceName province with :provinceCode code
+     */
+    public function iAddTheProvinceWithCode(string $provinceName, string $provinceCode): void
+    {
+        $this->client->addSubResourceData(
+            'provinces',
+            ['code' => $provinceCode, 'name' => $provinceName]
+        );
+    }
+
+    /**
+     * @When I add the :provinceName province with :provinceCode code and :provinceAbbreviation abbreviation
+     */
+    public function iAddTheProvinceWithCodeAndAbbreviation(string $provinceName, string $provinceCode, string $provinceAbbreviation): void
```

Comment:
I would go with variables without a `province` prefix - we're in the step that adds province, so it should be quite obvious

Reply:
Yes, i change.